src/toxcore/dht/client.rs

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
Maybe better move this file in dht/server/client.rs, while server.rs should be moved to dht/server/mod.rs?

Reply:
Good idea.